packages/discord.js/src/structures/VoiceState.js

```diff
@@ -206,28 +206,55 @@ class VoiceState extends Base {
   }
 
   /**
-   * Toggles the request to speak in the channel.
-   * Only applicable for stage channels and for the client's own voice state.
-   * @param {boolean} [request=true] Whether or not the client is requesting to become a speaker.
-   * @example
-   * // Making the client request to speak in a stage channel (raise its hand)
-   * guild.me.voice.setRequestToSpeak(true);
-   * @example
-   * // Making the client cancel a request to speak
-   * guild.me.voice.setRequestToSpeak(false);
-   * @returns {Promise<void>}
+   * Data to edit the logged in user's own voice state with, when in a stage channel
+   * @typedef {Object} VoiceStateEditData
+   * @property {boolean} [requestToSpeak] Whether or not the client is requesting to become a speaker.
+   * @property {boolean} [suppressed] Whether or not the user should be suppressed.
```

Comment:
```suggestion
   * @property {boolean} [requestToSpeak] Whether the client is requesting to become a speaker.
   * @property {boolean} [suppressed] Whether the user should be suppressed.
```

Reply:
I don't really feel like changing this. It's correct and I just copied the old docs, didn't edit them at all